egui/src/ui.rs

```diff
@@ -883,6 +883,25 @@ impl Ui {
         self.add(TextEdit::multiline(text))
     }
 
+    /// A `TextEdit` for code editing.
+    ///
+    /// Se also [`TextEdit::code_editor`].
```

Comment:
```suggestion
    /// A `TextEdit` for code editing.
    ///
    /// This will be multiline, monospace, and will insert tabs instead of moving focus.
    ///
    /// See also [`TextEdit::code_editor`].
```